docs/02-membership.md

```diff
@@ -0,0 +1,350 @@
+3. In a roughly full-time capacity. Anything less receives a [partial weighting](https://protocol-guild.readthedocs.io/en/latest/02-membership.html#time-weight-formula).
+
+## 2.3 Adding a Member
+
+An existing member should make a PR at [this repo](https://github.com/protocolguild/documentation) which proposes an addition to the member list above, along with the accompanying info:
+
+- Name / Identifier
+- Affiliated project they work on which is eligible
+- Links to relevant eligible work, eg. GitHub, research
+- 3-4 sentence summary of their contributions
+
+See some examples here: [open](https://github.com/protocolguild/documentation/pulls) or [closed](https://github.com/protocolguild/documentation/pulls?q=is%3Apr+is%3Aclosed+label%3A%22add+member%22) PRs
+
+Discussion should be open for at least one week to give members time to review and discuss.
+
+Bias or conflicts of interest of the nominator should be disclosed, if they exist, e.g. where one is an advisor to the other’s side project. After one week of rough consensus, the PR should be merged or rejected.
+
+## 2.4 Removing a Member
+
+### Self-removal, affiliation, weights
+
+Removals from the active membership should come from the member themselves, i.e. self-removal. Where this isn't possible due to extenuating circumstances, the member should be notified or tagged on the PR so they are aware of the changes.
+
+Affiliation and weight changes should include some rationale for the change, ideally from the member themselves and seconding by a colleague.
+
+### Forced-removal
+
+This situation occurs when another member proposes the removal of an existing member - even if they are continuing with eligible work. This should only happen in special circumstances where the cost of contention is higher than the loss of institutional legitimacy which their continued membership would produce.
+
+The PR should include ample references to the alleged misconduct and justification for the removal. To date, this method has never been used.
+
+## 2.5 Time-Weight Formula
+
+The mechanism uses a formula with member-specific inputs to produce a time-weight:
+
+`time_weight` = `SQRT`((`start_date` - `months_inactive`) * `full_or_part_time`)
+
+Each member's time-weight is updated onchain every quarter along with an Ethereum address they control to allocate the funding flowing through the mechanism. Let's explore each of the formula components below.
+
+1. `time_weight`
+
+Our formulation recognizes the local knowledge contributors gain over time, and uses that as a proxy for "value to the commons" and to allocate funding to members. Existing contributor weights get "diluted" as newcomers show up. Continuing contributors get additional weight per month they are active.
```

Comment:
I would move this to the top and be more descriptive here, e.g. `time_weight` is a floating point number representing the square root of the time a member has been contributing to Ethereum, in $UNIT